index.html

```diff
@@ -1243,21 +1243,48 @@ <h2>Evidence</h2>
         <h2>Disputes</h2>
 
         <p>
-The time may come when an <a>entity</a> wants dispute a <a>credential</a>
-issued by another entity. The mechanism for doing this is the same as issuing
-a regular credential except that the <a>subject</a> identifier for the
-<a>claims</a> are those of the disputed credential. For example, if a disputed
-credential with an identifier of <code>https://example.org/credentials/245</code>
-contains disputed statements, an entity would issue the following credential
-in a public venue to make it known that the credential is disputed:
-        </p>
+The time may come when an <a>entity</a> wants to dispute a <a>credential</a>
+issued by the issuer. There at least two different cases to consider:
+a subject disputes a claim made by the issuer, e.g. the address property is out
+of date, or an entity disputes a (false) claim made by the issuer about a different subject, e.g. an imposter is claiming the entity's social security number. 
+Only the subject of a verifiable credential, or its authorised
+agent, is entitled to issue a "DisputeCredential". A "DisputeCredential" issued by anyone other than the subject, or its authorised agent, SHOULD be disregarded by the verifier, unless the verifier has some out of band means of ascertaining the truth of
```

Comment:
Perhaps instead of saying who is entitled to issue a "DisputeCredential" we should say that verifiers should only accept those issued by the subject? We say this in the next sentence, so perhaps just drop the one about entitlement?

Also, w3c specs use American English spelling so "authorised" => "authorized".

Reply:
Agreed. I think we get in trouble when talking about "authorized agent" because we then have to explain what one is. Also, yes... W3C specs are in American English (for better or worse).

Reply:
I have removed authorized agent as requested.